qiskit_experiments/library/heat/zx_heat.py

```diff
@@ -0,0 +1,60 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+"""
+HEAT experiments for ZX Hamiltonian.
```

Comment:
I think the original motivation for separation is reusability. I don't think these analysis classes are reusable (probably wrong design)

Reply:
Maybe it should be "HEAT experiments and analysis for ZX Hamiltonian" if we generally keep the experiment and analysis separate? In plain English, just "experiments" makes sense but in the context of this repo where we usually separate experiment and analysis it might be confusing.

Reply:
Done in https://github.com/Qiskit/qiskit-experiments/pull/460/commits/149b59879a185c3809577dedee256e947c80b92a

Probably it's good time to reconsider the separation rule. I don't find any advantage of separating files of the dedicated analysis from its experiment. But this should be separate PR.